web-console/src/lib/functions/felderaRelation.ts

```diff
@@ -0,0 +1,28 @@
+import { Field, Relation } from '$lib/services/manager'
+import invariant from 'tiny-invariant'
+
+export type CaseDependentName = {
+  case_sensitive?: boolean | undefined
+  name: string
+}
+
+export const quotifyRelationName = (relation: Relation) =>
+  relation.case_sensitive ? `"${relation.name}"` : relation.name
+
+export const caseDependentName = (quotedRelationName: string) => {
+  const name = /\"?(\w+)\"?/.exec(quotedRelationName)?.[1]
+  invariant(name, `Invalid relation name parsed: ${name}`)
```

Comment:
Why so complicated? You can just use startsWith and endsWith, and then strip them.
This is overkill, and harder to read.
Please keep the code as simple as possible.

Reply:
Replaced with a simple ```const name = quotedRelationName.replaceAll('"', '')```
Generally I prefer to use regex if more than one String method is needed otherwise